scripts/fabricstat

```diff
@@ -115,6 +128,24 @@ class FabricPortStat(FabricStat):
             cnstat_dict[port_name] = PortStat._make(cntr)
         return cnstat_dict
 
+    def save_fresh_stats(self):
+        # Get stat for each port and save
+        counter_port_name_map = self.db.get_all(self.db.COUNTERS_DB, COUNTERS_FABRIC_PORT_NAME_MAP)
+        if counter_port_name_map is None:
```

Comment:
>counter_port_name_map [](http://example.com/codeflow?start=11&length=21)

I do not see how `counter_port_name_map` is used in this function. Is it really needed. Do you want to print some info before return?

Reply:
print a message before the return.